packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
+            "endpointName": {}
+          }
+        }
+      }
+    },
+    "nodeSettings": {
+      "providerIdShort": {
+        "__regexp": "^[a-fA-F0-9]{7}$"
+      },
+      "nodeVersion": {
+        "__regexp": "^([0-9]*\\.){2}([0-9]*)$"
+      },
+      "cloudProvider": {
+        "__regexp": "^aws$"
+      },
+      "region": {},
+      "stage": {},
+      "logFormat": {
+        "__regexp": "^(json|plain)$"
+      },
+      "chains": {
+        "__arrayItem": {
+          "id": {},
+          "type": {
+            "__regexp": "^evm$"
+          },
+          "providers": {
+            "__arrayItem": {
+              "name": {},
+              "url": {
+                "__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$"
+              },
+              "__optional": {
+                "blockHistoryLimit": {},
+                "minConfirmations": {}
+              }
+            }
+          },
+          "__optional": {
+            "providerAdminForRecordCreation": {
+              "__regexp": "^0x[a-fA-F0-9]{40}$"
+            },
+            "contracts": {
+              "__keyRegexp": "^(Airnode|Convenience)$",
+              "__objectItem": {
+                "__regexp": "^0x[a-fA-F0-9]{40}$"
+              }
+            }
+          }
+        }
+      }
+    },
+    "id": {
+      "__conditions": [
+        {
+          "__if": {
+            "__this": ".*"
+          },
+          "__rootThen": {
+            "security": {
+              "id": {
+                "__regexp": "^__match$",
+                "__level": "error"
+              }
+            }
+          }
+        }
+      ]
+    }
```

Comment:
Since `id` is a required field in `config.json`, I think it would make more sense to use your `__require` condition. Would it be difficult to add a functionality that looks like this?
```
"__require": {
    "/security.id": __this
}
```

or something like that ...

Reply:
It most certainly wouldn't be something difficult, however im deciding if I should remove `__require` altogether. During the development of validator `__if` conditions became quite powerful and can also do everything that `__require` can do, which made `__require` redundant. Currently `__require` is useful just because of it's compact syntax and has way too many downsides. Few of them would be:

- Its another keyword with its own rules that needs to be documented and remembered by whoever reads/writes templates
- If new feature is implemented into `__if`, for consistency sake it must be implemented, documented, tested in `__require` as well
- Conditions are generally buggy spots in validator, I fixed multiple nightmare bugs in them, this adds more potential buggy places

There are some more minor reasons, but you get the idea, so for now I would recommend not to use `__require` and after the issues in templates are resolved, i will analyze how much could `__require` potentially help and decide to keep/remove it.